hevm.cabal

```diff
@@ -185,11 +187,13 @@ library
     split                             >= 0.2.3 && < 0.3,
     template-haskell                  >= 2.19.0 && < 3,
     extra                             >= 1.7.14 && < 2,
+    hspec                             >= 2.9.0 && < 2.13.0,
```

Comment:
I am getting a warning from the compiler that these packages are not needed.
This is config the `hevm` library, right? The CLI changes should not be affecting the library, no?

Reply:
Nice catch, as always! Sorry, we disabled these warnings to become errors, because of an annoying warning that can't be fixed (` - system-cxx-std-lib-1.0 (exposed by flag -package-id system-cxx-std-lib-1.0`) and so I just normalized these warnings. Well-spotted!

Fixed :) I am merging, it seems to be working for my checks. I have read and re-read the code a few times, it seems fine to me.